tests/functional/test_post_company.py

```diff
@@ -49,6 +49,31 @@ def test_post_valid_company(client, get_token):
     assert result['year'] == 1969
 
 
+def test_post_valid_company_optional_fields(client, get_token):
+    '''
+    GIVEN a Flask application configured for testing
+    WHEN a POST request is made to /api/v1/companies with a valid company missing some optional fields
+    THEN check that the response is valid and particular data can be found
+    '''
+    data = {
+        "city_name": "Hoorn",
+        "company_name": "Test company #777",
```

Comment:
Maybe a good idea to leave some required field out (after all, we should be able to only send a change in company_size for instance.